src/Components/Pages/Plot/JSRootGraph.js

```diff
@@ -31,7 +30,18 @@ function createMultigraphFromDataSeries(energies, dataSeries) {
         ? JSROOT.createTMultiGraph(...filtered)
         : JSROOT.createTGraph(1)
 
-    if (res) res.fTitle = ""
+    if (res) {
+        res.fTitle = ''
+        const hist = JSROOT.createHistogram("TH1F", 20)
+        hist.fXaxis.fTitle = 'Energy[MeV/nucl]'
+        hist.fXaxis.fXmin = 1e-3
+        hist.fXaxis.fXmax = 1e+4
+        hist.fYaxis.fTitle = `Stopping power[${stpUnit.name}]`
+        hist.fYaxis.fXmin = 1e-4
+        hist.fYaxis.fXmax = 1e+2
+        console.log(hist)
+        res.fHistogram = hist
```

Comment:
@piotrpolec could you please center the labels using the method described here:
https://github.com/root-project/jsroot/issues/225#issuecomment-998748035
```
h1.fXaxis.InvertBit(JSROOT.EAxisBits.kCenterTitle)
```